models/restaurant.js

```diff
@@ -47,23 +53,25 @@ class Restaurant {
         //console.log("location:", location, city, postal_code, loc, restaurantName, OM_API_KEY)
         const result = await axios.get(`https://openmenu.com/api/v2/location.php?key=${OM_API_KEY}&country=us&${location}=${loc}&s=${restaurantName}`)
             .catch((err) => {
-                //console.log(err)
+                console.log(err)
```

Comment:
REQUIRED: Remove console logs in your commits. Use Javascript Console Utils to help with making and removing them.

Reply:
Removed